2025-01-13 10:48:23.052014: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:485] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2025-01-13 10:48:23.488125: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:8454] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2025-01-13 10:48:23.622253: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1452] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2025-01-13 10:48:24.839846: I tensorflow/core/platform/cpu_feature_guard.cc:210] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2025-01-13 10:49:37.466196: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
Traceback (most recent call last):
  File "/home/ymbahram/projects/def-hadi87/ymbahram/improved-diffusion/improved_diffusion/SDFT/evaluation/avg_SDFT_gammas_datasets_lpips.py", line 5, in <module>
    import evaluation 
  File "/lustre06/project/6086105/ymbahram/improved-diffusion/improved_diffusion/SDFT/evaluation/evaluation.py", line 36, in <module>
    vgg_model = VGG16(weights=VGG_PATH, include_top=False)
  File "/lustre06/project/6086105/ymbahram/gputorch3_10/lib/python3.10/site-packages/keras/src/applications/vgg16.py", line 96, in VGG16
    raise ValueError(
ValueError: The `weights` argument should be either `None` (random initialization), 'imagenet' (pre-training on ImageNet), or the path to the weights file to be loaded.  Received: weights=/home/ens/AT74470/util_files/vgg16_weights_tf_dim_ordering_tf_kernels_notop.h5
